src/api.ts

```diff
@@ -1,7 +1,12 @@
-import { OpenseaAsset } from './types/OpenseaAsset';
+import { OpenseaAssetsAndNextCursor } from './types/OpenseaAsset';
 
 export const OPENSEA_API_OFFSET = 50;
+const OPENSEA_URL = 'https://api.opensea.io';
+const ENS_GRAPH_URL = 'https://api.thegraph.com/subgraphs/name/ensdomains/ens';
+const MAX_AUTO_RETRY_ATTEMPT = 20;
```

Comment:
Setting auto retry to the max indicated in the prop comment.

```suggestion
const MAX_AUTO_RETRY_ATTEMPT = 10;
```